extensions/value_generators/models/generators.py

```diff
@@ -31,7 +30,7 @@ class Copier(value_generators_domain.BaseValueGenerator):
 
     default_value = ''
 
-    def generate_value(self, context_params, value, parse_with_jinja=False):
+    def generate_value(self, context_params, value, parse_with_jinja=False):  # pylint: disable=unused-argument
```

Comment:
If we don't support jinja then why do we need to have this argument? `parse_with_jinja=False`

Reply:
Did a soft removal, but yea doesn't make sense in keeping it. Removed it. 

Done.

Reply:
keeping `parse_with_jinja` in kwargs since ParamChange object should have that in datastore. But ignoring it in the generators. [Ref](https://github.com/oppia/oppia/pull/12757/files/505bba050549b8d2fe031946ec53f328f10988ba#r629885652)